src/Uno.UI/UI/Xaml/Markup/Reader/XamlStringParser.cs

```diff
@@ -32,7 +32,7 @@ public XamlFileDefinition Parse(string content)
 			// Force the line info, otherwise it will be enabled only when the debugger is attached.
 			var settings = new XamlXmlReaderSettings() { ProvideLineInfo = true };
 
-			using (var reader = new XamlXmlReader(document, context, settings))
+			using (var reader = new XamlXmlReader(document, context, settings, (_, _) => new KeyValuePair<bool?, bool>(null, false)))
```

Comment:
Reading it again, I wonder, why KeyValuePair, instead of a tuple or a record for instance? It would make the two booleans easier to grasp.

Reply:
@jeromelaban Tuple not available on net461. I guess you will be moving away from net461 for testing soon, so we could refactor it later?

Reply:
Let's make it a class then for now. It'll make refactoring easier.

Reply:
It is, thanks.